examples/debian_snapshot_all_arch/bullseye_all.yaml

```diff
@@ -0,0 +1,20 @@
+# Packages for examples/debian_snapshot_all_arch.
```

Comment:
Can you move this test into [apt/tests/resolution](https://github.com/GoogleContainerTools/rules_distroless/tree/main/apt/tests/resolution) folder? trying to keep examples helpful